src/Esfa.Vacancy.Register.Infrastructure/Services/ApprenticeshipSearchService.cs

```diff
@@ -50,10 +50,10 @@ public ApprenticeshipSearchService(IProvideSettings provideSettings, ILog logger
                     .Type("apprenticeship")
                     .Skip((parameters.PageNumber - 1) * parameters.PageSize)
                     .Take(parameters.PageSize)
-                    .Query(q => q.Filtered(
-                            sl => sl.Filter(
-                                fs => fs.Terms(
-                                    f => f.SubCategoryCode, parameters.SubCategoryCodes)))));
+                    .Query(query =>
+                        query.Terms(f => f.SubCategoryCode, parameters.SubCategoryCodes)
+                        && query.Range(p => p.OnField(r => r.PostedDate).GreaterOrEquals(parameters.FromDate))
+                    ));
```

Comment:
Shouldn't the search allow for:
1) Search by SubCategoryCodes only
2) Search by PostedDate only
3) Search by SubCategoryCodes and PostedDate

Reply:
Per Story (Mark), SubCategory either framework or standard is must. Posted date is optional.